doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,343 @@
+Visit :ref:`ref_make_container` for more information on how
+to create your own MAPDL docker image.
+
+Once you have created and uploaded your Docker image to a registry,
+you can start to pull and use the image in other devices.
+
+.. warning::
+
+   MAPDL Docker images are not allowed to be shared in
+   public or free-to-access repositories or registries.
+   Doing this you violate Ansys policy.
+
+
+
+Configure Docker to access a private GitHub registry
+----------------------------------------------------
+
+In case you have created a Docker image and upload it to a GitHub
+private repository, you must authorise your docker installation
+to access this private package by using a personal access
+token.
+Create a GitHub personal access token with ``packages read`` permissions
+according to `Creating a personal access token <gh_creating_pat_>`_
+
+Save that token to a file with:
+
+.. code::
+
+   echo MY_GITHUB_TOKEN_WITH_PACKAGE_READ_PERMISSION > GH_TOKEN.txt
+
+
+This lets you send the token to Docker without leaving the token value
+in your history. Next, authorize Docker to access this repository
+with:
+
+.. code::
+
+    GH_USERNAME=<my-github-username>
+    cat GH_TOKEN.txt | docker login ghcr.io -u $GH_USERNAME --password-stdin
+
+
+.. _run_an_mapdl_image:
+
+Run an MAPDL Docker image
+=========================
+
+You can now launch MAPDL from Docker using the command line, a
+`docker compose file <run_an_mapdl_image_using_docker_compose_>`_ 
+or a script that gather the commands given to the command line.
+
+Your Docker image should have a valid MAPDL license configuration.
+The easiest way is to have an environment variable :env:`ANSYSLMD_LICENSE_FILE`
+pointing to a valid license server. This environment variable can be already
+included in the Docker image. However this is not recommended because it could
+expose the license server if the Docker image get leaked.
+The recommended approach is to set that environment variable when running the
+container. 
+
+To instantiate an MAPDL Docker container from an image hosted at ``ghcr.io/myuser/myrepo/mymapdldockerimage`` use:
+
+.. code:: bash
+
+  ANSYSLMD_LICENSE_FILE=1055@MY_LICENSE_SERVER_IP
+  LOCAL_MAPDL_PORT=50053
+  MAPDL_DOCKER_REGISTRY_URL=ghcr.io/myuser/myrepo/mymapdldockerimage
+  docker run -e ANSYSLMD_LICENSE_FILE=$ANSYSLMD_LICENSE_FILE --restart always --name mapdl -p $LOCAL_MAPDL_PORT:50052 $MAPDL_DOCKER_REGISTRY_URL -smp > log.txt
+
+For Windows use:
+
+.. code:: pwsh-session
+
+    $env:ANSYSLMD_LICENSE_FILE="1055@MY_LICENSE_SERVER_IP"
+    $env:LOCAL_MAPDL_PORT=50053
+    $env:MAPDL_DOCKER_REGISTRY_URL="ghcr.io/myuser/myrepo/mymapdldockerimage"
+
+    docker run -e ANSYSLMD_LICENSE_FILE=$env:ANSYSLMD_LICENSE_FILE --restart always --name mapdl -p $env:LOCAL_MAPDL_PORT`:50052   $env:MAPDL_DOCKER_REGISTRY_URL -smp
+
+
+First time you run it, Docker logins into the registry and
+pulls the required image.
+This can take some time depending on the size of the image.
```

Comment:
```suggestion
The first time you instantiate the container, Docker logins into the registry and
pulls the required image. This can take some time, depending on the size of the image.
```